src/index.js

```diff
@@ -124,17 +124,18 @@ export default class ModalFilterPicker extends Component {
   renderOption = (rowData) => {
     const {
       selectedOption,
-      renderOption
+      renderOption,
+        optionTextStyle
     } = this.props
 
     const { key, label } = rowData
 
     let style = styles.optionStyle
-    let textStyle = styles.optionTextStyle
+    let textStyle = optionTextStyle||styles.optionTextStyle
 
     if (key === selectedOption) {
       style = styles.selectedOptionStyle
-      textStyle = styles.selectedOptionTextStyle
+      textStyle = optionTextStyle||styles.selectedOptionTextStyle
```

Comment:
Would be better to have another prop called `selectedOptionTextStyle` rather than re-use the normal style.